pkgs/development/libraries/bashup-events/default.nix

```diff
@@ -0,0 +1,6 @@
+{ callPackage }:
+
+{
+  bashup-events32 = callPackage ./3.2.nix { };
```

Comment:
and this file can be removed with this.

Reply:
I tried this when I was splitting them out previously, but it doesn't look like nixpkgs-review follows the pattern:
```console
$ nix run nixpkgs#nixpkgs-review rev HEAD
...
$ nix-env -f /Users/abathur/.cache/nixpkgs-review/rev-a4c89401cfb87a1c70a0731577fece7c62752d9e/nixpkgs -qaP --xml --out-path --show-trace --meta
Nothing to be built.
$ nix-shell /Users/abathur/.cache/nixpkgs-review/rev-a4c89401cfb87a1c70a0731577fece7c62752d9e/shell.nix
```

I know it would work, but 2 assumptions led me to keep the current form:
1. nixpkgs-review compatibility will pay better ongoing dividends than whatever efficiency comes from dropping the extra file
2. if it confuses nixpkgs-review, it might also confuse ofborg

Reply:
That should work but whatever.